Commerce 10634 2 Fake

iovaneale opened the pull request.

sergiojimcos committed:
LPS-154730 Add GET, DELETE and UPSERT methods by siteId and externalReferenceCode in rest-openapi

sergiojimcos committed:
LPS-154730 buildREST

sergiojimcos committed:
LPS-154730 Implement GET, DELETE and UPSERTS methods in DocumentFolderResourceImpl

sergiojimcos committed:
LPS-154730 Implement new methods to ADD, GET and FETCH folder by ERC in service modules

sergiojimcos committed:
LPS-154730 Update Folder models

sergiojimcos committed:
LPS-154730 Modify DLFolder import process

sergiojimcos committed:
LPS-154730 Update addFolder methods in other test cases from different modules

sergiojimcos committed:
LPS-154730 buildService

sergiojimcos committed:
LPS-154730 baseline

sergiojimcos committed:
LPS-154730 Update addFolder methods in other modules

Commit:
LPS-160224 Support automatic translation of web content with DeepL

sergiojimcos committed:
LPS-154730 Modify JSONFolder for externalReferenceCode field

adolfopa committed:
LPS-160224 SF

Commit:
LPS-160224 Refactor

Commit:
LPS-160224 Build Lang

ces-totrinh committed:
LPS-166868 Remove DLOpenerFileEntryReference if the file is not found when deleting in Google Drive

ces-totrinh committed:
LPS-166868 Allow doing Checkin if file is not found when deleting in Google Drive

ces-totrinh committed:
LPS-166868 Add warning message

ces-totrinh committed:
LPS-166868 Allow editing in Google Drive if file is not found when deleting in Google Drive

noornajj committed:
LPS-169881 Add the impersonated user's id to the url in Liferay.Util.fetch

noornajj committed:
LPS-169881 Refactor all instances where doAsUserId is added to Liferay.Util.fetch

ces-totrinh committed:
LPS-166868 buildLang

oliverkecskemety committed:
LPS-173238 Use the admin set configuration for explicitConsentMode

For LPS-170904

noornajj committed:
LPS-169881 Define the method getDoAsUserIdEncoded in Liferay.ThemeDisplay to prevent test cases from failing

Commit:
LPS-173238 SF inline (companyId is always > 0)

For LPS-170904

Commit:
LPS-173238 SF simplify

For LPS-170904

Commit:
LPS-173238 Avoid getting system configuration when httpServletRequest is not null

For LPS-170904

oliverkecskemety committed:
LPS-173240 Check if Cookies preference handling is enabled before checking consent type.
If not enabled delete all remaining consent cookies from user.

For LPS-170904

oliverkecskemety committed:
LPS-173241 Remove no longer needed parts

For LPS-170904

sergiojimcos committed:
LPS-173137 Delete readOnly parentTaxonomyCategory's property

sergiojimcos committed:
LPS-173137 buildREST

sergiojimcos committed:
LPS-173137 Add support to PATCH and UPDATE by taxonomyVocabularyId parameter

sergiojimcos committed:
LPS-173137 Add support to PATCH and UPDATE by parentTaxonomyCategory

ericyanLr committed:
LPS-169881 Fix additional js tests

sergiojimcos committed:
LPS-175558 Add test cases for checking nested entities in OpenAPI

sergiojimcos committed:
LPS-175558 Add nested entities in OpenAPI schema for Custom Objects and System Objects that has got a M-M and 1-M relationship

brianchandotcom committed:
LPS-154730 Wordsmith

JanisZhang committed:
LPS-174310 Remove Stream usages in WabProcessor

sergiojimcos committed:
LPS-173137 Add test case for PATCH

JanisZhang committed:
LPS-174310 Remove Stream usages in WabProcessorTest

Commit:
LPS-174414 Replace stream with TransformUtil.transform()

Commit:
COMMERCE-10634 Add tests to Relevant and fix priorities

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

iovaneale commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/3515/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3515/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 27 out of 28 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 141 out of 147 jobs passed in 3 hours 7 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/faf682e989d8872bdea67d3b17c2d4318c888502">faf682e989d8872bdea67d3b17c2d4318c888502</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/db97ad35086d4f2e2bad8156f20d9fb49ab82117">db97ad35086d4f2e2bad8156f20d9fb49ab82117</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2183964614">EE Development Acceptance (master) - 4248 - 2023-02-01[20:14:24]</a></p><details><summary><strong>ci:test:stable - 27 out of 28 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li>central-requirements-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148394/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366663/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/155717/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205968/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162332/">js-unit-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159421/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408638/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205971/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396200/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/158871/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346069/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407615/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429824/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251273/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366662/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137319/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205966/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450592/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396197/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251272/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396199/"><strike>modules-integration-mysql57-jdk8_stable/0/1</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346068/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366661/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429819/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407609/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205974/">rest-builder-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143262/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396202/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>27 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 140 out of 147 jobs PASSED</strong></summary><div><h4>7 Failed Jobs:</h4><ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/3515/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148394/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450598/">empty-osgi-core-dir-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157868/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366663/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/156570/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407610/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162333/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429820/">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429821/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407613/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148636/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424027/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408635/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/154245/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137320/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370248/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162940/">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148424/">functional-tomcat90-mysql57-chrome100-jdk8/0/11</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429822/">functional-tomcat90-mysql57-chrome100-jdk8/0/12</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/132558/">functional-tomcat90-mysql57-chrome100-jdk8/0/13</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408637/">functional-tomcat90-mysql57-chrome100-jdk8/0/14</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/158870/">functional-tomcat90-mysql57-chrome100-jdk8/0/15</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162334/">functional-tomcat90-mysql57-chrome100-jdk8/0/16</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205970/">functional-tomcat90-mysql57-chrome100-jdk8/0/17</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407614/">functional-tomcat90-mysql57-chrome100-jdk8/0/18</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408636/">functional-tomcat90-mysql57-chrome100-jdk8/0/19</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125842/">functional-tomcat90-mysql57-chrome100-jdk8/0/20</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366660/">functional-tomcat90-mysql57-chrome100-jdk8/0/21</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143963/">functional-tomcat90-mysql57-chrome100-jdk8/0/22</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157870/">functional-tomcat90-mysql57-chrome100-jdk8/0/23</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205969/">functional-tomcat90-mysql57-chrome100-jdk8/0/24</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147642/">functional-tomcat90-mysql57-chrome100-jdk8/1/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450593/">functional-tomcat90-mysql57-chrome100-jdk8/1/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205967/">functional-tomcat90-mysql57-chrome100-jdk8/1/2</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424025/">functional-tomcat90-mysql57-chrome100-jdk8/1/3</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429817/">functional-tomcat90-mysql57-chrome100-jdk8/1/4</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407612/">functional-tomcat90-mysql57-chrome100-jdk8/1/5</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370246/">functional-tomcat90-mysql57-chrome100-jdk8/1/6</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/161841/">functional-tomcat90-mysql57-chrome100-jdk8/1/7</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396198/">functional-tomcat90-mysql57-chrome100-jdk8/1/8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148635/">functional-tomcat90-mysql57-chrome100-jdk8/1/9</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157742/">functional-tomcat90-mysql57-chrome100-jdk8/1/10</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429816/">functional-tomcat90-mysql57-chrome100-jdk8/1/11</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/161840/">functional-tomcat90-mysql57-chrome100-jdk8/1/12</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/132557/">functional-tomcat90-mysql57-chrome100-jdk8/1/13</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407611/">functional-tomcat90-mysql57-chrome100-jdk8/1/14</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346067/">functional-tomcat90-mysql57-chrome100-jdk8/1/15</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251271/">functional-tomcat90-mysql57-chrome100-jdk8/1/16</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143962/">functional-tomcat90-mysql57-chrome100-jdk8/1/17</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157869/">functional-tomcat90-mysql57-chrome100-jdk8/1/18</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366659/">functional-tomcat90-mysql57-chrome100-jdk8/1/19</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429815/">functional-tomcat90-mysql57-chrome100-jdk8/1/20</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157741/">functional-tomcat90-mysql57-chrome100-jdk8/1/21</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/161839/">functional-tomcat90-mysql57-chrome100-jdk8/1/22</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162938/">functional-tomcat90-mysql57-chrome100-jdk8/1/23</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137739/">functional-tomcat90-mysql57-chrome100-jdk8/1/24</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205975/">functional-tomcat90-mysql57-chrome100-jdk8/2/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159978/">functional-tomcat90-mysql57-chrome100-jdk8/2/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157744/">functional-tomcat90-mysql57-chrome100-jdk8/2/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408641/">functional-tomcat90-mysql57-chrome100-jdk8/2/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148426/">functional-tomcat90-mysql57-chrome100-jdk8/2/4</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162942/">functional-tomcat90-mysql57-chrome100-jdk8/2/5</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251275/">functional-tomcat90-mysql57-chrome100-jdk8/2/6</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424031/">functional-tomcat90-mysql57-chrome100-jdk8/2/7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125844/">functional-tomcat90-mysql57-chrome100-jdk8/2/8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147645/">functional-tomcat90-mysql57-chrome100-jdk8/2/9</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143964/">functional-tomcat90-mysql57-chrome100-jdk8/2/10</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143263/">functional-tomcat90-mysql57-chrome100-jdk8/2/11</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159422/">functional-tomcat90-mysql57-chrome100-jdk8/2/12</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450597/">functional-tomcat90-mysql57-chrome100-jdk8/2/13</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251274/">functional-tomcat90-mysql57-chrome100-jdk8/2/14</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407617/">functional-tomcat90-mysql57-chrome100-jdk8/2/15</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/340605/">functional-tomcat90-mysql57-chrome100-jdk8/2/16</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137321/">functional-tomcat90-mysql57-chrome100-jdk8/2/17</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148637/">functional-tomcat90-mysql57-chrome100-jdk8/2/18</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/154246/">functional-tomcat90-mysql57-chrome100-jdk8/2/19</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/161842/">functional-tomcat90-mysql57-chrome100-jdk8/2/20</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408639/">functional-tomcat90-mysql57-chrome100-jdk8/2/21</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/132559/">functional-tomcat90-mysql57-chrome100-jdk8/2/22</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/125843/">functional-tomcat90-mysql57-chrome100-jdk8/2/23</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/157743/">functional-tomcat90-mysql57-chrome100-jdk8/2/24</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/155718/">functional-tomcat90-mysql57-chrome100-jdk8/3/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147644/">functional-tomcat90-mysql57-chrome100-jdk8/3/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424029/">functional-tomcat90-mysql57-chrome100-jdk8/3/2</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370249/">functional-tomcat90-mysql57-chrome100-jdk8/3/3</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450596/">functional-tomcat90-mysql57-chrome100-jdk8/3/4</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137740/">functional-tomcat90-mysql57-chrome100-jdk8/3/5</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162335/">functional-tomcat90-mysql57-chrome100-jdk8/3/6</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205972/">functional-tomcat90-mysql57-chrome100-jdk8/3/7</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429823/">functional-tomcat90-mysql57-chrome100-jdk8/3/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162941/">functional-tomcat90-mysql57-chrome100-jdk8/3/9</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147643/">functional-tomcat90-mysql57-chrome100-jdk8/3/10</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429818/">functional-tomcat90-mysql57-chrome100-jdk8/3/11</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450595/">functional-tomcat90-mysql57-chrome100-jdk8/3/12</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424026/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159976/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450594/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159420/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/155717/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205968/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162332/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-jdk8<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159977/">lpkg-controller-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/159421/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408638/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205971/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396200/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/158871/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346069/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407615/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429824/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251273/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366662/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/137319/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205966/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/450592/">modules-compile-jdk8/0/12</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396197/">modules-compile-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408642/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/383450/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366664/"><strike>modules-integration-mysql57-jdk8/0/2</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346071/"><strike>modules-integration-mysql57-jdk8/0/3</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/408640/">modules-integration-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205973/"><strike>modules-integration-mysql57-jdk8/0/5</strike></a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396203/"><strike>modules-integration-mysql57-jdk8/0/6</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407618/"><strike>modules-integration-mysql57-jdk8/0/7</strike></a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424030/">modules-integration-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346070/">modules-integration-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407616/">modules-integration-mysql57-jdk8/0/10</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396201/">modules-integration-mysql57-jdk8/0/11</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370250/">modules-integration-mysql57-jdk8/0/12</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429825/">modules-integration-mysql57-jdk8/0/13</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/251272/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396199/"><strike>modules-integration-mysql57-jdk8_stable/0/1</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346068/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429826/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370251/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366661/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/424028/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/383449/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-jdk8<ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/162939/">patching-tool-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/429819/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407609/">portal-license-smoke-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205974/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/148425/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/143262/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/370247/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396202/">unit-jdk8_stable/0/0</a></li></ul></li></ul></div><details><summary><strong>140 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3515/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at db97ad35086d4f2e2bad8156f20d9fb49ab82117:</strong></summary><ol><li><div><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366664//consoleText">modules-integration-mysql57-jdk8/0/2</a><ol><li><details><summary><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366664/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366664//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346071//consoleText">modules-integration-mysql57-jdk8/0/3</a><ol><li><details><summary><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346071/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/346071//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205973//consoleText">modules-integration-mysql57-jdk8/0/5</a><ol><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205973/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205973//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396203//consoleText">modules-integration-mysql57-jdk8/0/6</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396203/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/396203//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407618//consoleText">modules-integration-mysql57-jdk8/0/7</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407618/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/407618//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li>...</li></ol></details></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/3515/">test-portal-acceptance-pullrequest(master)#3515</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3515/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/iovaneale/liferay-portal/pull/37">iovaneale#37</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2185287658">[master] ci:test:relevant - iovaneale &gt; iovaneale - PR#37 - 2023-02-03[07:47:56]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-35.liferay.com/job/publish-testray-report/8055/">publish-testray-report#8055</a><br/></div></div>

4lejandrito committed:
LPS-173137 Testing also the patch to parentAssetCategory

4lejandrito committed:
LPS-173137 SF

brianchandotcom committed:
LPS-117666 Inline

brianchandotcom committed:
LPS-117666 Use transformToArray

brianchandotcom committed:
LPS-117666 Sort

JanisZhang committed:
LPS-174709 Remove Stream usages in KeyLocalizedLabelPairCommaSeparatedLabelsInfoTextFormatter

JanisZhang committed:
LPS-174709 Remove Stream usages in AssetPublisherDisplayContext

ces-totrinh committed:
LPS-166515 Remove previous lar files

istvansajtos committed:
LPS-117666 Move test to testGetUserAccount

istvansajtos committed:
LPS-117666 Rename

haydenzheng314 committed:
LPS-174920 Remove Stream<> usage in KaleoDesignerPortlet

oliverkecskemety committed:
LPS-173241 Adjust CookiesPreAction tests

For LPS-170904

oliverkecskemety committed:
LPS-173241 Better edge case handling

For LPS-170904

oliverkecskemety committed:
LPS-173241 Add new integration tests for CookiesManagerImpl

For LPS-170904

haydenzheng314 committed:
LPS-174920 Use JSONUtil, inline jsonArray

JanisZhang committed:
LPS-175361 Remove Stream usages in JournalContentDisplayContext

sergiojimcos committed:
LPS-154730 Delete unnessary _validateExternalReferenceCode method

kresimir-coko committed:
COMMERCE-10474 Add support for comma and enter in quickAddToCart

JLucasM committed:
COMMERCE-10454 Fix the bug that not updating the form submit when I change the checkbox

ling-alan-huang committed:
LPS-175154 Move logic to checkstyle, because SF is not smart enough to do typecast

ling-alan-huang committed:
LPS-175154 Add documentation

ling-alan-huang committed:
LPS-175154 SF

ling-alan-huang committed:
LPS-175154 SF, use the method references

sergiojimcos committed:
LPS-173137 Rename

Commit:
LPS-173241 We never need to mutate this map (nor allow it) so we can use a constant.

For LPS-170904

peerkar committed:
LPS-175508: Move code to portal-search

peerkar committed:
LPS-175508 portal-search: Switch to RTE

peerkar committed:
LPS-175508 portal-search: Don't break all the suggestions when a single source fails

peerkar committed:
LPS-175508 portal-search: Set enabled

sandrodw3 committed:
LPS-175670 Add shortcut to select/deselect multiple items

We are wrapping the checkbox in a new component so it's unknown for the ClayTreeView and it doesn't overwrite its state and event handlers

lorenzolapescara committed:
COMMERCE-10799 Fix missing variables in macros due to poshi issue

sandrodw3 committed:
LPS-175670 Update count feedback height

sandrodw3 committed:
LPS-175670 Apply also to layout tree

sandrodw3 committed:
LPS-175670 Add item count to layout tree as well

sandrodw3 committed:
LPS-175670 buildLang

Commit:
COMMERCE-10634 Adapt integers to new poshi rules

JanisZhang committed:
LPS-175361 Simplify

ces-totrinh committed:
LPS-166515 Remove lar file when the changes have been successfully propagated from Site Template

IstvanD committed:
LPS-175701 Adding language key for user notification configuration

IstvanD committed:
LPS-175701 buildLang

IstvanD committed:
LPS-175701 Adding UserNotificationConfigurationCategory

IstvanD committed:
LPS-175701 Adding dependencies

IstvanD committed:
LPS-175701 Adding UserNotificationConfiguration

IstvanD committed:
LPS-175701 Adding scheduled task that deletes all archived user notification events older than a given number of days.

Commit:
LPS-175847 Update DDMField#MULTIPLE_LINES Path

Commit:
LPS-175847 Fix viewWithStructureCP macro

Commit:
LPS-175864 Update WCEditWebContent#TEXT_INPUT path

Commit:
LPS-175919 Add more tests on open API profile

22LucaGiannotta22 committed:
COMMERCE-10474 Add functional tests and change a previous one

sergiojimcos committed:
LPS-175515 Check if a permission is null to not retrieve it in the API response

AliciaGarciaGarcia committed:
LPS-175127 obtain the maxNumberOfPages and pass to the creation of the pdf previews on PDFBox

sergiojimcos committed:
LPS-175515 buildREST

AliciaGarciaGarcia committed:
LPS-175153 obtain the maxNumberOfPages and pass to the creation of the pdf previews on ghostscript

ealonso committed:
LPS-175999 Adds support for DLFileEntryTypeKey as itemSubtype

Brunno0liveira committed:
COMMERCE-10454 Add functional test CanAssertProductSourceUpdated

haydenzheng314 committed:
LPS-175668 Remove Stream<> usage in PortalPreferencesUserSegmentsEntrySimulator

haydenzheng314 committed:
LPS-174920 SF, inline method

JoyceWang08211 committed:
LRQA-79275 Extend target

JoyceWang08211 committed:
LRQA-79275 Add macro for custom db

haydenzheng314 committed:
LPS-175668 Use Long::parseLong instead

haydenzheng314 committed:
LPS-175926 Replace Stream<> usage with for loop

DiegoHu97 committed:
LPS-176048 Replace icons in application-list-taglib

AliciaGarciaGarcia committed:
LPS-175313 Revert "LPS-172530 Rename"

AliciaGarciaGarcia committed:
LPS-175313 Revert "LPS-172530 SF"

AliciaGarciaGarcia committed:
LPS-175313 Revert "LPS-172530 Add test integration"

AliciaGarciaGarcia committed:
LPS-175313 Revert "LPS-172530 Update test"

AliciaGarciaGarcia committed:
LPS-175313 Revert "LPS-172530 Don't show the searchFolder in the result."

ealonso committed:
LPS-175999 Rename to fileEntryTypeKey

ealonso committed:
LPS-175999 Return null if it doesn't exist

sergiojimcos committed:
LPS-175558 Fix some consistency code and add Description to the Schema from M part of a 1-M relationship

Jeremy-Ch committed:
LPS-175902 Update paths for the input box

DiegoHu97 committed:
LPS-176048 Replace icons in layout-admin-web

DiegoHu97 committed:
LPS-176048 Replace icons in site-navigation-admin-web

marcosapmf committed:
LRAC-12719 sync custom field in the firstSync

dsitu committed:
COMMERCE-10840 Convert commerceAccountType to accountEntryType

ealonso committed:
LPS-175999 Simplify

ling-alan-huang committed:
LPS-175586 Update packages `@clayui/*` to v3.88.0

ling-alan-huang committed:
LPS-169220 Auto SF

ling-alan-huang committed:
LPS-141138 Auto SF

ling-alan-huang committed:
LPS-141138 SF, inline

ling-alan-huang committed:
LPS-175154 Reword

brianchandotcom committed:
LPS-117666 SF

brianchandotcom committed:
LPS-166868 SF

brianchandotcom committed:
LPS-166868 Can you rephrase this? It doesn't make any sense. Thx.

brianchandotcom committed:
LPS-166868 Regen

ruben-pulido committed:
LPS-176107 Transform scopeIds into groupIds when creating a Collection based on an Asset Publisher widget configuration

liferay-translation committed:
LPS-77699 Update Translations

ling-alan-huang committed:
LPS-176109 Enforce variable and method naming for type Criteria and RuleNode

ling-alan-huang committed:
LPS-176109 Rename

Commit:
LPS-175847 Modify locators to adapt new markup

JoyceWang08211 committed:
LRQA-79567 Update macro

Commit:
LPS-176115 Add test to add content page template in publication and publish

JoyceWang08211 committed:
LRQA-79275 Update test suite for custom DB

JoyceWang08211 committed:
LRQA-79275 SF

ealonso committed:
LPS-176119 We should keep this logic removed in LPS-175547

Commit:
LPS-176025 Add test to create page with existing page template

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-176026 Add test to create custom fragments within publication

22LucaGiannotta22 committed:
COMMERCE-10861 Change path

jkappler committed:
LPS-176153 Make sure that permission checker is initialized

Mikellorza committed:
LPS-176138 Fix

ccorreagg committed:
LPS-176172 Avoid overriding if isSecure is false

victorg1991 committed:
LPS-175670 Keep the outer div so the react component doesn't destroy the parent

gaborlovasdotcom committed:
LRQA-79568 Update test OIDC app details

gaborlovasdotcom committed:
LRQA-79568 Re-activate test case

ealonso committed:
LPS-176119 Adds supports in ItemSelectorViewDescriptor to multiple selection

ealonso committed:
LPS-176119 When we are setting the multipleSelection to the url it is lost if we navigate through group and asset libraries selector, so, instead of setting the multipleSelection to the url, use the new method.

sergiojimcos committed:
LPS-173137 Check if the taxonomyVocabularyId from the body is null and is different from the assetCategory that we are going to update or patch

DiegoHu97 committed:
LPS-176048 Replace alert with clay:alert in orphan_portlets.jsp

joseabelenda committed:
LPS-176175 Create SiteInitializerConfiguration

joseabelenda committed:
LPS-176175 Add siteInitializer to client-extension.properties

ruben-pulido committed:
LPS-175972 Add portlet-configuration-web.json with link to learn page

ruben-pulido committed:
LPS-175972 Adapt language key

ruben-pulido committed:
LPS-175972 buildLang

ruben-pulido committed:
LPS-175972 buildGradle

ruben-pulido committed:
LPS-175972 Use liferay-learn taglib instead of hardcoding URL on JSP

Commit:
LPS-176173 Provide a ServletRequestWrapper to isolate the HttpSession from AuthVerifier authenticated requests

Commit:
LPS-169817 Re-invoke the AuditFilter after authenticating a request with AuthVerifierFilter

Commit:
LPS-176173 Isolate all portal session attributes unless the request is authenticated with PortalSessionAuthVerifier

ccorreagg committed:
LPS-176172 Add test to validate the path and protocol is correct based on the configuration

brianchandotcom committed:
LPS-176107 SF

brianchandotcom committed:
LPS-176175 Fix SF

brianchandotcom committed:
LPS-176175 Should it be groupName or siteName? I'm fine with either.

brianchandotcom committed:
LPS-176175 prep next

victorg1991 committed:
LPS-176179 Use correct type options

brianchandotcom committed:
LPS-175701 Wordsmith

brianchandotcom committed:
LPS-175701 SF

brianchandotcom committed:
LPS-175515 prep next

brianchandotcom committed:
LPS-175515 prep next

Commit:
LPS-176173 SemVer

oliverkecskemety committed:
LPS-173241 Sort

For LPS-170904

jkappler committed:
LPS-176193 Move documents to folder and adapt page definitions

WYZ-Issac committed:
LPS-169615 Add a dependency file for translate emedded tags

WYZ-Issac committed:
LPS-168615 Update the xml file to copy and update dependency translation file

WYZ-Issac committed:
LPS-168615 Update the macro to copy dependency file

WYZ-Issac committed:
LPS-168615 Add a test to ensures that user can import translation with embedded bpt/ept tags

JanisZhang committed:
LPS-175361 Remove Optional usages in JournalArticleDDMFormFieldValueRequestParameterRetriever

JanisZhang committed:
LPS-175361 Remove Stream usages in JournalArticleDDMTemplateInfoItemTemplatedRenderer

Commit:
LPS-176021 Modify locator to make it reusable

Hijky committed:
LPS-175083 Add a macro to subscribe structure content folder

Hijky committed:
LPS-175083 Add a macro to create user as a administrator

Hijky committed:
LPS-175083 Add a macro to create a object action to a object

Hijky committed:
LPS-175083 Add a folder to include user notification testcase and macro

Hijky committed:
LPS-175083 Update a macro to set up global user account Id

Hijky committed:
LPS-175083 Update the properties file to add a new testcase file

hhtank committed:
LPS-166515 Deprecate class BackgroundTaskContextMapConstants in module level, add new class in portal-kernel *.constants

hhtank committed:
LPS-166515 baseline

Commit:
LPS-176020 Modify case to adapt new behavior

dantewang committed:
LPS-174310 SF

JessieZhao0714 committed:
LPS-176208 Update locator

JanisZhang committed:
LPS-174709 Remove Stream usages in AssetEntriesCheckerHelper

JanisZhang committed:
LPS-174709 SF

Commit:
LPS-176112 Modify case to adapt new behavior

Commit:
LPS-176015 Add step to wait for changes saved

JoyceWang08211 committed:
LRQA-79565 Prepare audit configuration for MFA test cases

Commit:
LPS-174780 Add case to view include child categories when searching parent categories instance setting

ealonso committed:
LPS-176119 baseline

ealonso committed:
LPS-176119 Allows define whether cet is multiple selection or not

ealonso committed:
LPS-176119 Update attending if it should be multiple selection or not

Commit:
LPS-174780 Add step to return categoryId

Commit:
LPS-174780 Add case to disable and enable include child categories when searching parent categories

liferay-translation committed:
LPS-77699 Update Translations

jkappler committed:
LPS-174405 Adds new language key

brianchandotcom committed:
LPS-175926 SF

brianchandotcom committed:
LPS-176138 SF

jkappler committed:
LPS-174405 Build lang

brianchandotcom committed:
LPS-173241 Use ReflectionUtil

jkappler committed:
LPS-174405 Use correct message

brianchandotcom committed:
LPS-170904 SF

brianchandotcom committed:
LPS-154730 Please pay this technical debt

brianchandotcom committed:
LPS-154730 Follow sql/portal-tables.sql #sfme

brianchandotcom committed:
LPS-154730 Ewe, but good enough

brianchandotcom committed:
LPS-154730 Auto SF

brianchandotcom committed:
LPS-173137 SF / Add TODO for tests

brianchandotcom committed:
LPS-175558 SF

brianchandotcom committed:
LPS-160224 Use newer libraries

brianchandotcom committed:
LPS-160224 DeepL is their official name

brianchandotcom committed:
LPS-160224 Rename

brianchandotcom committed:
LPS-160224 Components slow things down. Let's only pay for it when it helps.

adolfopa committed:
LPS-174913 Check for specific values, not ranges

Negative values should behave as any other invalid value instead of
silently returning an empty RSS feed.  Let the services throw the
relevant exceptions for each case.

brianchandotcom committed:
LPS-160224 Why is DeepLTranslatorConfigurationModelListener#onBeforeSave checking 2 strings and this is checking 3 strings? Please resend this snippet.

brianchandotcom committed:
LPS-160224 Inline

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-160224 Simplify

4lejandrito committed:
LPS-176080 Autogenerating create batch actions test

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 I think your logic was wrong. So I read up on https://www.deepl.com/docs-api/general/ and I believe the param is 'type'

4lejandrito committed:
LPS-176080 buildREST

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 We're already passing the authKey as a part?

4lejandrito committed:
LPS-176080 Overriding _getExpectedActions for reported issues pending to be resolved

    Reported issues that are still not fixed: LPS-174609, LPS-175141, LPS-175142, LPS-175138, LPS-175139, LPS-175838, LPS-175146, LPS-175145,LPS-175147, LPS-175837, LPS-175840, LPS-174610. The Form Record case has not been reported as the module will be soon deprecated.
testGetCountryRegionsPage_getExpectedActions has also been overridden as Batch Actions not yet implemented for Regions.

Co-authored-by: Magdalena Jedraszak <magdalena.jedraszak@liferay.com>

brianchandotcom committed:
LPS-160224 According the docs, one is GET and the other is POST

brianchandotcom committed:
LPS-160224 Better name

brianchandotcom committed:
LPS-160224 Better name

brianchandotcom committed:
LPS-160224 Clarity

brianchandotcom committed:
LPS-160224 Throw an exception instead

brianchandotcom committed:
LPS-160224 Simplify

brianchandotcom committed:
LPS-176080 SF

brianchandotcom committed:
LPS-176080 Sort

brianchandotcom committed:
LPS-176080 Wordsmith

brianchandotcom committed:
LPS-176080 SF/Wordsmith

brianchandotcom committed:
LPS-176080 prep next

brianchandotcom committed:
LPS-176080 prep next

brianchandotcom committed:
LPS-166515 Sort

brianchandotcom committed:
LPS-166515 Update usages

brianchandotcom committed:
LPS-166515 auto SF
